ffi/tests/ctests/build.rs

```diff
@@ -0,0 +1,11 @@
+extern crate cc;
+
+fn main() {
+    cc::Build::new()
+        .include("../../include")
+        .file("src/tests.c")
+        .flag("-Wall")
```

Comment:
Please use built-in cc methods, otherwise you're basically asking for non-portable flags :) (e.g. `-Wextra` fails with MSVC)

From `cc` docs:
![image](https://user-images.githubusercontent.com/557590/55246277-cea5d280-523c-11e9-98ec-0ecb2a792631.png)

Reply:
Fixed.

Reply:
Actually, according to docs the `warnings` and `extra_warnings` are enabled by default, but I guess it's okay to be explicit here.